### PR TITLE
Enabled termination protection for serverless demo cloudformation stack

### PR DESCRIPTION
**Pull request by anuranduttaroy**
**Issue #:** NA

**Description of changes:**
Update deploy script to enable termination protection for all cloudformation stacks for serverless demo.

**Testing**

1. Have you successfully run `npm run build:release` locally?
Yes.

2. How did you test these changes?
Tried running the deploy script locally and verifying the stack is being created and termination protection is enabled.

3. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?
NA

4. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?
NA

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: demos/serverless/deploy.js [hunk 1]
```diff
@@ -164,6 +166,7 @@ console.log('Deploying serverless application');
 spawnOrFail('sam', ['deploy', '--template-file', './build/packaged.yaml', '--stack-name', `${stack}`,
                     '--parameter-overrides', `UseEventBridge=${useEventBridge} ChimeEndpoint=${chimeEndpoint}`,
                     '--capabilities', 'CAPABILITY_IAM', '--region', `${region}`, '--no-fail-on-empty-changeset']);
+spawnOrFail('aws', ['cloudformation', 'update-termination-protection', '--enable-termination-protection', '--stack-name', `${stack}`], null, false);
```
Comment (ltrung) [inline demos/serverless/deploy.js:169]: Why do we need to disable printout for this?
Also, is update-termination-protection something we need to do every time we deploy?

File: demos/serverless/deploy.js [hunk 2]
```diff
@@ -101,7 +106,9 @@ function spawnOrFail(command, args, options) {
     process.exit(255);
   }
   const output = cmd.stdout.toString();
-  console.log(output);
+  if(printOutput) {
```
Comment (simmkyu) [inline demos/serverless/deploy.js:109]: nit: add a space after `if`, e.g. `if (printOutput) {`

File: demos/serverless/deploy.js [hunk 3]
```diff
@@ -164,6 +171,9 @@ console.log('Deploying serverless application');
 spawnOrFail('sam', ['deploy', '--template-file', './build/packaged.yaml', '--stack-name', `${stack}`,
                     '--parameter-overrides', `UseEventBridge=${useEventBridge} ChimeEndpoint=${chimeEndpoint}`,
                     '--capabilities', 'CAPABILITY_IAM', '--region', `${region}`, '--no-fail-on-empty-changeset']);
+if(enableTerminationProtection) {
```
Comment (simmkyu) [inline demos/serverless/deploy.js:174]: `if (`

**Comment by simmkyu** [comment]
I see that `--enable-termination-protection` is `false`. Please make sure that the new option `--enable-termination-protection` works well when redeploying the CF stack.

**Comment by anuranduttaroy** [comment]
> I see that `--enable-termination-protection` is `false`. Please make sure that the new option `--enable-termination-protection` works well when redeploying the CF stack.

Yes, I tested this by deploying multiple times.

File: demos/serverless/deploy.js [hunk 4]
```diff
@@ -18,12 +19,13 @@ const packages = [
...
-  console.log(`  -a, --application    Browser application to deploy, default '${app}'`);
-  console.log(`  -e, --event-bridge   Enable EventBridge integration, default is no integration`);
-  console.log(`  -c, --chime-endpoint AWS SDK Chime endpoint, default is '${chimeEndpoint}'`);
+  console.log(`  -r, --region                         Target region, default '${region}'`);
+  console.log(`  -b, --s3-bucket                      S3 bucket for deployment, required`);
+  console.log(`  -s, --stack-name                     CloudFormation stack name, required`);
+  console.log(`  -a, --application                    Browser application to deploy, default '${app}'`);
+  console.log(`  -e, --event-bridge                   Enable EventBridge integration, default is no integration`);
+  console.log(`  -c, --chime-endpoint                 AWS SDK Chime endpoint, default is '${chimeEndpoint}'`);
+  console.log(`  -t, --enable-termination-protection  Enable termination protection for the Cloudformation stack, default is false`);
```
Comment (simmkyu) [inline demos/serverless/deploy.js:28]: nit: `Cloudformation` => `CloudFormation` but you can skip this

**Review by simmkyu — APPROVED** [review]

File: src/versioning/Versioning.ts [hunk 5]
```diff
@@ -24,7 +24,7 @@ export default class Versioning {
     // to generate `version.ts` is absent, and will be until someone runs
     // `publish` in this new world.
     /* istanbul ignore next */
-    return VERSION.semverString || '1.19.14';
+    return VERSION.semverString || '1.19.15';
```
Comment (ltrung) [inline src/versioning/Versioning.ts:27]: This should not be needed anymore right?

**Review by simmkyu — APPROVED** [review]

File: src/versioning/Versioning.ts (hunk 5 above)
Comment (anuranduttaroy) [reply src/versioning/Versioning.ts:27]: Yeah I got that after reading the chat messages. But does not harm to have this change.

**Pull request merged**